backends/occa/ceed-occa-basis.c

```diff
@@ -125,6 +125,7 @@ static int CeedTensorContract_Occa(CeedInt A, CeedInt B, CeedInt C, CeedInt J,
       for (CeedInt j=0; j<J; j++) {
         CeedScalar tq = t[j*tstride0 + b*tstride1];
         for (CeedInt c=0; c<C; c++)
+          // NOLINTNEXTLINE: False positive for unitalized values
```

Comment:
I added `NOLINT` for the offending line. This is an uncommon false positive.

Reply:
Yeah, it thinks `A*J*C` could be non-positive, but asserting that that is not the case wasn't fixing it for me either. The NOLINT is fine by me.